packages/angular_devkit/schematics/tools/description.ts

```diff
@@ -25,6 +25,7 @@ export interface FileSystemSchematicJsonDescription {
   readonly factory: string;
   readonly description: string;
   readonly schema?: string;
+  readonly extends?: string;
```

Comment:
Both need to be there... unless the two types are not correctly deriving from the other.